features/step_definitions/assertions_steps.rb

```diff
@@ -4,4 +4,12 @@
 
 Then("I am logged into my account") do
 	expect(current_path).to eq welcome_index
-end
\ No newline at end of file
+end
+
+Then("I should see all my messages") do
```

Comment:
Should be something like 'Then I see that I am in my inbox' to relate to the path in the next 2 line
.

Reply:
Changed